src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx

```diff
@@ -52,24 +54,49 @@ export default class TeamSettings extends AsyncView {
 
   renderBody() {
     let team = this.props.team;
+    let {teamId, orgId} = this.props.params;
+
+    let access = new Set(this.context.organization.access);
 
     return (
-      <Form
-        model={this.model}
-        apiMethod="PUT"
-        saveOnBlur
-        allowUndo
-        onSubmitSuccess={this.handleSubmitSuccess}
-        onSubmitError={() => addErrorMessage(t('Unable to save change'))}
-        initialData={{
-          name: team.name,
-          slug: team.slug,
-        }}
-      >
-        <Box>
-          <JsonForm location={this.context.location} forms={teamSettingsFields} />
-        </Box>
-      </Form>
+      <React.Fragment>
+        <Form
+          model={this.model}
+          apiMethod="PUT"
+          saveOnBlur
+          allowUndo
+          onSubmitSuccess={this.handleSubmitSuccess}
+          onSubmitError={() => addErrorMessage(t('Unable to save change'))}
+          initialData={{
+            name: team.name,
+            slug: team.slug,
+          }}
+        >
+          <Box>
+            <JsonForm location={this.context.location} forms={teamSettingsFields} />
+          </Box>
+        </Form>
+
+        {access.has('team:admin') && (
+          <div className="box">
```

Comment:
oops I missed this the first time but if copy this it'll give us a more consistent UI.

https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/organization/general/organizationGeneralSettingsView.jsx#L144

Reply:
Like, use `<Panel/>` and `<PanelHeader/>`, etc?

Reply:
Ya and `Field`